cypress/integration/fee-fine/fee-fine-history.spec.js

```diff
@@ -0,0 +1,139 @@
+import moment from 'moment';
+import permissions from '../../support/dictionary/permissions';
+import TestTypes from '../../support/dictionary/testTypes';
+import ServicePoints from '../../support/fragments/settings/tenant/servicePoints/servicePoints';
+import UserEdit from '../../support/fragments/users/userEdit';
+import Users from '../../support/fragments/users/users';
+import UsersOwners, { getNewOwner } from '../../support/fragments/settings/users/usersOwners';
+import TransferAccounts, { getNewTransferAccount } from '../../support/fragments/settings/users/transferAccounts';
+import ManualCharges from '../../support/fragments/settings/users/manualCharges';
+import PaymentMethods from '../../support/fragments/settings/users/paymentMethods';
+import WaiveReasons, { getNewWaiveReason } from '../../support/fragments/settings/users/waiveReasons';
+import RefundReasons, { getNewRefundReason } from '../../support/fragments/settings/users/refundReasons';
+import NewFeeFine from '../../support/fragments/users/newFeeFine';
+import AppPaths from '../../support/fragments/app-paths';
+import FeeFinesDetails from '../../support/fragments/users/feeFineDetails';
+import PayFeeFaine from '../../support/fragments/users/payFeeFaine';
+import AddNewStaffInfo from '../../support/fragments/users/addNewStaffInfo';
+import { HTML } from '../../../interactors';
+
+//The test fails because of the bug.
+describe('Fee/Fine history ', () => {
+  const userData = {
+    permissions: [
+      permissions.uiUsersView.gui,
+      permissions.uiUsersfeefinesCRUD.gui,
+      permissions.uiUsersViewServicePoints.gui,
+      permissions.uiUsersfeefinesView.gui,
+      permissions.uiUsersManualCharge.gui,
+      permissions.uiUsersManualPay.gui,
+      permissions.uiUserAccounts.gui
+    ],
+  };
+  const ownerData = {};
+  let feeFineType = {};
+  const paymentMethod = {};
+  const waiveReason = {};
+  const refundReason = {};
+  const transferAccount = getNewTransferAccount();
+  let feeFineAccount;
+  let servicePointId;
```

Comment:
I don't see the necessity in this action because it is not convenient to have a large block  ServicePoints.getViaApi till the end of the before section of the describe block. Also it is convenient to see the list of all variables in one place.